nuclia/sdk/kbs.py

```diff
@@ -17,7 +17,11 @@ def _auth(self) -> NucliaAuth:
         return auth
 
     @accounts
-    def list(self, account: Optional[str] = None):
+    @zone
+    def list(self, account: Optional[str] = None, zone: Optional[str] = "europe-1"):
+        if not zone:
+            raise ValueError("zone is required")
+
```

Comment:
```suggestion
    @zone
    @accounts
    def list(self, account: Optional[str] = None, **kwargs):
```
To keep code consistency on how we overwrite params in other places I would suggest having just **kwargs.
Also I think it is better to keep both decorators, @accounts and @zone, one doesn't replace the other